Sluggo/Models/Managers/UserManager.swift

```diff
@@ -44,4 +44,12 @@ class UserManager {
         
         JsonLoader.executeCodableRequest(request: requestBuilder.getRequest(), completionHandler: completionHandler)
     }
+    
+    public func getRecord(identity: AppIdentity, completionHandler: @escaping(Result<AuthRecord, Error>) -> Void) -> Void {
+        let requestBuilder = URLRequestBuilder(url: URL(string: identity.baseAddress + "auth/user/")!)
+            .setMethod(method: .GET)
+            .setIdentity(identity: identity)
+        
+        JsonLoader.executeCodableRequest(request: requestBuilder.getRequest(), completionHandler: completionHandler)
```

Comment:
How is this any different from getUser()? This appears to do the same thing save for passing the identity as a alarm instead of using the instance identity